src/projectscene/qml/Audacity/ProjectScene/clipsview/TrackClipsItem.qml

```diff
@@ -340,4 +339,45 @@ Item {
         anchors.bottom: parent.bottom
         thickness: 2
     }
+
+    Connections {
+        target: root.container
+
+        function onClipMoveRequested(clipKey, completed) {
+            // this one moves the clips
+            let verticalMoveSuccess = clipsModel.moveSelectedClips(clipKey, completed)
+
+            // this one notifies every ClipListModel about moveActive
+            root.updateMoveActive(completed)
+
+            // clip might change its' track, we need to update grabbed clipKey
+            if (verticalMoveSuccess) {
+                setHoveredClipKey(clipsModel.updateClipTrack(clipKey));
+            }
+        }
+    }
+
+    Connections {
+        target: root.container
+
+        function onClipStartEditRequested(clipKey) {
```

Comment:
you can use multiple functions for one target, so no need to create Connections for each function